packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -257,11 +266,6 @@ export interface PopoverProps {
    * Additional data that is dispatched with the tracking event.
    */
   tracking?: TrackingProps;
-  /**
-   * Displace the floating element from its core placement along specified axes.
-   * @see https://floating-ui.com/docs/offset
```

Comment:
Removed the ref to the Floating UI docs since we obstruct a part of Floating UI's `offset` middleware (i.e. we don't expose the middleware function). I replaced it with a bit more copy in the jsdoc description of the prop.